### PR TITLE
Update doc

### LINKED ISSUE #381: Update doc, pkgdown, ...
TO DO:

- [x] update pkgdown website with new shinylive examples. TO DO: issue with plot layer block initialisation (passed data from the previous block should be the real data and not the plot object).
- [x] update vignettes to fix breaking changes.
- [x] Remove Posit Connect deploy workflow (as we use shinylive).
- [x] BUMP NEWS.md + version number: 0.2.0?

### PR DESCRIPTION
**Pull request by DivadNojnarg**
Fix #381, #377.

### CODE REVIEW

**Comment by DivadNojnarg** [comment]
@nbenn Took he freedom to move linting step to the first CICD check as this is so frustrating to fail a pipeline after 10 min just for a whitespace.
Screenshot 2024-06-06 at 22 32 50

**Comment by DivadNojnarg** [comment]
@nbenn Ready for review. I edited the `news.md` and the version number to 0.0.2 as the old one was 0.0.1.9000. Let me know if I missed elements in this `news.md`.

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+select_blk <- new_select_block("STUDYID")
+
+stack <- new_stack(
+  data_blk,
+  select_blk
+)
+
+# Old way
+stack <- new_stack(
+  new_ data_block,
```
Comment (nbenn) [inline NEWS.md:32]: Typo.

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+- Evaluation stops whenever a block isn't valid and the app should not crash.
+- We can now instantiate block outside the stack with default parameter values (or use the old way with constructors):
+
+```r
+# New way
+data_blk <- new_dataset_block(selected = "lab", package = "blockr.data")
+select_blk <- new_select_block("STUDYID")
+
+stack <- new_stack(
+  data_blk,
```
Comment (nbenn) [inline NEWS.md:26]: You could also inline, the ctor calls.

**Review by nbenn — APPROVED** [review]
Lgtm, thanks!

**Pull request merged**